### ISSUE TITLE
A possible bug in deepspeed configuration - zero3_offload.json

### ISSUE THREAD

**Issue by yushuinanrong** [haotian-liu/LLaVA#1433]
### Describe the issue

Hi, 

First of all, great work!
I noticed in the zero3_offload.jon config, it has set the learning rate scheduler to WarmupLR ([this line](https://github.com/haotian-liu/LLaVA/blob/main/scripts/zero3_offload.json#L22)), and it will override the input scheduler (cosine in pre-training and fine-tuning). In our experiments, it shows that WarmupLR will cause the loss hover around 1 and leads to poor performance. 

Thanks,
JC

**Comment by yqy2001** [comment]
+1. I delete the [`scheduler`](https://github.com/haotian-liu/LLaVA/blob/3e337ad269da3245643a2724a1d694b5839c37f9/scripts/zero3_offload.json#L22) field in [zero3_offload.json](https://github.com/haotian-liu/LLaVA/blob/main/scripts/zero3_offload.json), which makes the lr scheduler bahave properly
